src/tests/catwalk_tests/test_predictors.py

```diff
@@ -150,33 +168,43 @@ def test_predictor_composite_index():
             'metta-uuid': '1234',
             'indices': ['entity_id', 'as_of_date'],
         }
+        # Creates a matrix entry in the Matrices table with uuid from metadata above
+        MatrixFactory(matrix_uuid = "1234")
+        session2.commit()
+
         matrix_store = InMemoryMatrixStore(matrix, metadata)
-        predict_proba = predictor.predict(
-            model_id,
-            matrix_store,
-            misc_db_parameters=dict(),
-            train_matrix_columns=['feature_one', 'feature_two']
-        )
 
-        # assert
-        # 1. that the returned predictions are of the desired length
-        assert len(predict_proba) == 4
+        # Runs the same test for training and testing predictions
+        for mat_type in ("train", "test"):
+            predict_proba = predictor.predict(
+                model_id,
+                matrix_store,
+                misc_db_parameters=dict(),
+                train_matrix_columns=['feature_one', 'feature_two'],
+                matrix_type = mat_type
```

Comment:
We follow PEP8, which says keyword argument assignments like this should have no space around the = sign